_internals/software-architecture.md

```diff
@@ -84,50 +64,25 @@ will learn which ones are important and what their purpose is.
 
 Here is a more detailed description of these extensions:
 
-#### `babelfishpg_common`: Microsoft SQL Server data types
+#### `babelfishpg_common`: SQL Server data types
 
 Data types are often really similar but not necessarily 100% identical.
 Therefore, the Babelfish development team provides specific data types which
-mimic Microsoft SQL Server behavior. Fortunately, there were only a handful of
-types which behave differently. Here is a complete list of these types:
-
-- `sys.BBF_BINARY`: Data type for binary data.
-- `sys.BIT`: Converts 1 or 0 to `t` or `f`.
-- `sys.BPCHAR` and `sys.VARCHAR`: Microsoft SQL Server specific `varchar` implementation
-- `sys.DATETIME`: A simple data type for time and date. For more precision use
-  `DATETIME2`.
-- `sys.DATETIME2`: An extension for `DATETIME` with more
-  precision and functionality.
-- `sys.DATETIMEOFFSET`: Like `DATETIME2`, but with timezone awareness.
-- `sys.SMALLDATETIME`: Defines a date that is combined with a time of day. The time
-  is based on a 24-hour day, with seconds always zero (:00) and without
-  fractional seconds.
-- `sys.SQL_VARIANT`: Enables these database objects to support values of
-  other data types.
-- `sys.UNIQUEIDENTIFIER`: 16 byte GUID (e.g. `6F9619FF-8B86-D011-B42D-00C04FC964FF`)
-
-#### `babelfishpg_money`: Fixed precision numeric type
-
-In addition to the core data types outlined in the previous section, there is
-one more data type which is packaged into a separate extension for licensing
-reasons: `fixeddecimal`.
-
-See the
-[documentation of the original software](https://github.com/2ndQuadrant/fixeddecimal/blob/master/README.md)
-for usage instructions and other information.
+mimic Microsoft SQL Server behavior. You will find an up-to-date list of unsupported 
```

Comment:
remove 'Microsoft'

Reply:
There is now one 'Microsoft' on the page, at the top...